src/ZoneServer/Network/PacketHandler.cs

```diff
@@ -2964,5 +2964,23 @@ public void CZ_CLIENT_DIRECT(IZoneConnection conn, Packet packet)
 
 			Send.ZC_CLIENT_DIRECT(character, type, argStr);
 		}
+
+
+		/// <summary>
+		/// Removes a buff from the player character.
+		/// </summary>
+		/// <param name="conn"></param>
+		/// <param name="packet"></param>
+		[PacketHandler(Op.CZ_BUFF_REMOVE)]
+		public void CZ_BUFF_REMOVE(IZoneConnection conn, Packet packet)
+		{
+			var buffId = (BuffId)packet.GetInt();
+			var character = conn.SelectedCharacter;
+
+			if (character.TryGetBuff(buffId, out var buff) && buff.Data.Removable)
```

Comment:
Are you sure? I think ignoring it is ok for now.
I think if we should deal with cheaters on another way, maybe creating a system to log it on a specific table?
Well it can get complex but I will leave the warning log then.

Reply:
We're currently not trying to catch cheaters, because we don't have enough information for that. We're leaving log messages so we can learn more and _potentially_ notice faulty implementations _or_ that someone is trying to cheat.